spec/core/client/v2/user_spec.rb

```diff
@@ -10,4 +10,10 @@
     expect(user).to be_a Core::Client::V2::User
     expect(user.fullName).to eq "Mike Potter"
   end
+
+  it ".current returns an existing user" do
+    user = Core::Client::V2::User.current.first
```

Comment:
Is there a way to avoid the `first` here? Similar to https://github.com/ableco/core-client-rb/blob/master/lib/core/client/v2/base.rb#L13

Reply:
Thanks for the suggestion!